src/main.rs

```diff
@@ -245,7 +245,7 @@ async fn main() -> Result<(), Box<dyn Error>> {
     ]);
 
     if let Some(c) = opt.command {
-        nu_cli::run_pipeline_standalone(c, opt.stdin, &mut context, true).await?;
+        nu_cli::run_script_file(c, true).await?;
```

Comment:
Less than confident these are the correct commands.

Reply:
from the naming it makes sense